### ISSUE TITLE
Pet Portraits stays on screen bug

### ISSUE THREAD

**Issue by Sadee1533** [Centias/BankItems#26]
It has taken me a long time to figure out what was causing the issue of a Battle pet portrait staying on my screen anywhere on my screen. I go to Gbank, mouse over something in the Gbank and a pet portrait comes on my screen and takes a good minute to fade off the screen. I go to the AH and looking at Battle pets in the AH and if I mouse over a pet its portrait is stuck on my screen.  If I turn off Bankitems it quits doing this. I have Pettracker, Rematch, daily pet timers, battle pet breeds for other addons so not sure if one of them is interfering with Bankitems or if its just Bankitems and Blizzard portraits of the Battle pets. Only way to stop this is to turn off the Bankitems on my toon that I sort, buy etc Battle pets. I don't use any other toons to work with Bpets . Ty so much for this addon by the way. If I wasn't disabled Id donate. I've been using this addon for years and years as I started in Vanilla. It took me a good hour to figure out how to get the files from Git to in my computer so it would work for my wow addons as I'm just not computer savvy lol. Turning off all other addons except Bankitems and this bug still happens. I have formatted and  only added Bankitems from here to the game for addons and it still happens. Im just not sure why it does it and I dont see anything in options that would make this happen. Bankitems only shows Caged pets in my bags and banks. It does not show or name the actual pet in the cage when looking through Gbank but it will still put a portrait up and stay on my screen

**Comment by Centias** [comment]
While I _have_ seen this happen before, it mostly happened in a specific situation, like if I open my mail box and I have a pet in unopened mail. However, I'm having an impossible time trying to figure out when and why it happens. So this may take a while to fix.

**Comment by vttale** [comment]
Ja, fwiw, it happens all the time with my auction alt.

**Comment by Sadee1533** [comment]
Yep  it happens with the AH for me also and the mail. The mail one is all the time. I buy and sell pets so as soon as I open the mail and havent even moused on the mail yet I have a pet portrait on my screenif there is a caged pet in the mail and  that will not go away for abit. Once it goes away the same pet portrit will come back again just going to the bank. I have to turn off bankitems and it will stop doing it.
